proto/regen/ecocredit/v1/events.proto

```diff
@@ -193,4 +194,19 @@ message EventBridge {
 
   // contract is the contract address.
   string contract = 3;
-}
\ No newline at end of file
+
+  // amount is the amount of credits.
+  string amount = 4;
```

Comment:
Added amount so that the bridge service receives all information needed in the event.